### ISSUE TITLE
Usage from a #![no_std] crate doesn't work

### ISSUE THREAD

**Issue by jonas-schievink** [panicbit/custom_debug_derive#6]
Deriving `CustomDebug` on a struct in a `#![no_std]` crate fails with this error:

```
error[E0433]: failed to resolve: could not find `std` in `{{root}}`
   --> rubble/src/link/advertising.rs:343:23
    |
343 | #[derive(Copy, Clone, CustomDebug)]
    |                       ^^^^^^^^^^^ could not find `std` in `{{root}}`
```

**Comment by panicbit** [comment]
Oh, I'll look into it. Give me a few hours!

**Comment by jonas-schievink** [comment]
No need, I already have a patch :)

**Comment by panicbit** [comment]
Fixed in #7 and 4f733b5a5
